python/activator/middleware_interface.py

```diff
@@ -532,7 +532,8 @@ def _export_refcats(self, center, radius):
         # collection, so we have to specify a list here. Replace this
         # with another solution ASAP.
         possible_refcats = ["gaia", "panstarrs", "gaia_dr2_20200414", "ps1_pv3_3pi_20170110",
-                            "atlas_refcat2_20220201", "gaia_dr3_20230707"]
+                            "atlas_refcat2_20220201", "gaia_dr3_20230707",
+                            "uw_stars_20240130", "uw_stars_20240228"]
```

Comment:
Do we actually need the `0130` version if we have access to `0228`?

Reply:
The `0130` version is what we currently have in the dev central repo.  And `0228` would be what we expect to use if we wanted `prod` today.   
I can copy the `0228` version into the dev central repo, and then `0130` wouldn't be needed here in this line.  But I hesitate to keep updating the dev repo just to get the newest data.

Reply:
That makes sense, but is `uw_stars_20240228` actually deployed (or expected to be deployed) in `/repo/embargo`?

Reply:
I don't know what will be added to `/reop/embargo` for the actual ops rehearsal. I'll remove `uw_stars_20240228` and wait to add the eventual refcat when that's known.

Reply:
On second thought, if the current `obs_lsst` default is `uw_stars_20240228`, copying the new refcat to pp's dev repo is probably better than a config override.  I'll check more.

Reply:
Change of mind.... 
`uw_stars_20240228` has now been imported to the dev repo. I changed the commit and use only `uw_stars_20240228`.